README.md

```diff
@@ -1 +1,7 @@
-"# Etch.OrchardCore.Liquid" 
+# Etch.OrchardCore.Liquid
```

Comment:
Let's flesh this out a lot more, see https://github.com/etchuk/etch.OrchardCore.SEO as an example. Should include build  status, how to install the module and what filters are available (with code example of how to use the filter). Small thing, you can remove the "A" from the description, "Collection of helpful liquid filters" is better.